EmbeddingTestBeds/Embedding.UWP/Embedding.UWP.csproj

```diff
@@ -11,9 +11,9 @@
     <AssemblyName>Embedding.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
```

Comment:
Why do we need to change the `TargetPlatformVersion`?.

Reply:
Hi @jsuarezruiz , I'm new to PRs for Xamarin. Forms, as for the TargetPlatformVersion they force me to change it because I no longer use 10.0.16299 .0.

Reply:
... and is ideal since the minimum version is <TargetPlatformMinVersion> 10.0.16299.0 </TargetPlatformMinVersion>